src/deck/drivers/src/flowdeck_v1v2.c

```diff
@@ -264,10 +264,23 @@ LOG_ADD(LOG_UINT8, squal, &currentMotion.squal)
 LOG_ADD(LOG_FLOAT, std, &stdFlow)
 LOG_GROUP_STOP(motion)
 
+/**
+ * Settings and parameters for handling of the flowdecks
+ * measurments
+ */
 PARAM_GROUP_START(motion)
+/**
+ * @brief Nonzero to turn on adaptive standard devivation estimation (default: 0)
+ */
 PARAM_ADD(PARAM_UINT8, disable, &useFlowDisabled)
```

Comment:
It seems disputed if this should be core or not? In the sheet you mark it as core. Should be discussed?

Reply:
Ah it was supposed to be a core one. It's a bit iffy though, since you don't want regular users to mess with this, but for many that doing research in estimators this is a very handy parameter. Technically I would say it is core. 

I also saw that I accidently switched the docs so I fixed that too